File: PPWC/Minor_Assignment_5/Q4.c
#include <stdio.h>
int main(){
    int a[3][3]={4,5,6,7,8,9,1,2,3};
    printf("%p %p %p\n",a[1]+2,*(a+1)+2,&a[1][2]);
    printf("%d %d %d\n",*(a[1]+2),*(*(a+1)+2), a[1][2]);
    return 0;
}

/*
Output--
0061FF10 0061FF10 0061FF10
9 9 9
*/

File: Python/Minor_Assignment_6/Q4.py
import numpy as np
array = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
array[[0, 1]] = array[[1, 0]]
array[:, [0, 1]] = array[:, [1, 0]]
print(array)

'''
Output--
[[5 4 6]
 [2 1 3]
 [8 7 9]]
'''